deps/los.lua

```diff
@@ -18,26 +18,67 @@ limitations under the License.
 
 --[[lit-meta
   name = "luvit/los"
-  version = "2.0.0"
+  version = "2.0.1"
   license = "Apache 2"
   homepage = "https://github.com/luvit/luvit/blob/master/deps/los.lua"
   description = "Tiny helper to get os name in luvit."
   tags = {"os"}
 ]]
 
-local jit = require('jit')
+local function get_os_name()
```

Comment:
Can we use `uv_os_uname`? It was added in https://github.com/luvit/luv/pull/313.

Reply:
Seems possible? but that should be used alongside the jit requirement, since `uv_os_name` was introduced in libuv 1.25 and older versions don't have it.

Reply:
I went ahead and added a check or uv_os_uname, but as Bilal mentioned it's Libuv 1.25+, so I've kept a call to the uname utility as a fallback in case everything else fails.